### PR TITLE
checkmarx 0.1.0

### PR DESCRIPTION
**Pull request by miguelfreitas93**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by miguelfreitas93** [comment]
Hi, 
Why I'm getting invalid commit hash ?

Thanks!

File: steps/checkmarx/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      is_required: true
+      is_sensitive: true
+  - cx_exclude_folders: 
+    opts:
+      title: Exclude Folders
+      description: |
+        Exclude Folders, for example: *test, *mock*, example*
+      is_expand: true
+      is_required: false
+      is_sensitive: true
```
Comment (koral--) [inline steps/checkmarx/0.1.0/step.yml]: This shouldn't be marked as sensitive perhaps.
Sensitive is mainly for credentials (passwords, access tokens etc.)

**Comment by koral--** [comment]
> Why I'm getting invalid commit hash ?

It seems that you have no tags in your git repo.

File: steps/checkmarx/0.1.0/step.yml (hunk 1 above)
Comment (miguelfreitas93) [reply steps/checkmarx/0.1.0/step.yml]: Hi Koral, 
You're right, I thought "is_sensitive" meant case sensitive.
Thanks !

**Comment by miguelfreitas93** [comment]
I have added the tag "0.1.0" and fixed the "is_sensitive" flag but build keeps failing. Does anyone know why ?

**Comment by koral--** [comment]
Still no tags on remote repo: https://github.com/miguelfreitas93/bitrise_checkmarx_step/tags
It seems that you forgot to push the tag.

**Comment by miguelfreitas93** [comment]
I had fixed the tag issue, but build keeps failing...
@koral-- can you please let me know what was the issue ?
The thing is, I'm working on Windows machine and I was not able to install Bitrise CLI due to Homebrew. I had tried Docker but I did not succeed. Since I don't have access to it, I'm trying to share the step manually following the guidelines

**Comment by lpusok** [comment]
`Invalid step: missing or empty required 'PublishedAt' property`
Please try to share using the officially reccomended way.

**Pull request closed without merging**